opennlp-docs/src/docbkx/doccat.xml

```diff
@@ -127,14 +127,14 @@ $ opennlp DoccatTrainer -model en-doccat.bin -lang en -data en-doccat.train -enc
 		<programlisting language="java">
 						<![CDATA[
 DoccatModel model = null;
-InputStream dataIn = null;
+InputStreamFactory dataIn = null;
 
-try (dataIn = new FileInputStream("en-sentiment.train")) {
+try (dataIn = new MarkableFileInputStreamFactory(new File("en-sentiment.train"))) {
```

Comment:
It didn't compile for me. I tried to move the declaration to a single statement, but that also didn't work. It complained that `InputStreamFactory` is not a closeable? Did you try this code @Alanscut ? I think we need some sort of input stream here, not the factory being open/closed?